pytorch_lightning/core/hooks.py

```diff
@@ -84,6 +85,18 @@ def on_train_end(self) -> None:
         """
         # do something at the end of training
 
+    def on_validation_start(self):
```

Comment:
just to be consistent :]
```suggestion
    def on_validation_start(self) -> None:
```

Reply:
ah thanks :+1: